pulsar-broker/src/test/java/org/apache/pulsar/broker/transaction/TransactionTest.java

```diff
@@ -167,4 +168,22 @@ public void testGetTxnID() throws Exception {
         Assert.assertEquals(txnID.getLeastSigBits(), 1);
         Assert.assertEquals(txnID.getMostSigBits(), 0);
     }
+
+    @Test
+    public void testSubscriptionRecreateTopic() throws PulsarAdminException, PulsarClientException {
+        String topicName = "persistent://pulsar/system/tests_SubscriptionRecreateTopic";
+
+        admin.topics().createNonPartitionedTopic(topicName);
+        admin.topics().unload(topicName);
+        try {
+            admin.topics().createNonPartitionedTopic(topicName);
```

Comment:
please add a "fail()" after this line, other wise if `createNonPartitionedTopic` succeeds then the test will pass